### ISSUE TITLE
Make error with Xavier

### ISSUE THREAD

**Issue by ChienLiu** [CaoWGG/TensorRT-CenterNet#33]
have someone successfully built with Nvidia Xavier or other Jetpack platforms?
I met a make error, Does someone have any suggestion?

this error is like that:

[ 46%] Linking CXX shared library ../../lib/libnvonnxparser.so
**/usr/bin/ld: cannot find -lcudart
/usr/bin/ld: cannot find -lcublas**
collect2: error: ld returned 1 exit status
onnx-tensorrt/CMakeFiles/nvonnxparser.dir/build.make:235: recipe for target '../lib/libnvonnxparser.so' failed
make[2]: *** [../lib/libnvonnxparser.so] Error 1
CMakeFiles/Makefile2:92: recipe for target 'onnx-tensorrt/CMakeFiles/nvonnxparser.dir/all' failed
make[1]: *** [onnx-tensorrt/CMakeFiles/nvonnxparser.dir/all] Error 2
Makefile:129: recipe for target 'all' failed
make: *** [all] Error 2

I know it is a error about cuda path,
but how can I modify the path cmake using?

thanks.

**Comment by meteorshowers** [comment]
you can communicate with me.
I am doing the same thing.
Wechat 16600286930
@ChienLiu

**Comment by facerless** [comment]
sudo ln -s /usr/local/cuda/lib64/libcudart.so /usr/lib/libcudart.so
It works for me！You can try

**Comment by ElonKou** [comment]
Maybe you can try in this issue #39

**Comment by zhaoxuyan** [comment]
> sudo ln -s /usr/local/cuda/lib64/libcudart.so /usr/lib/libcudart.so
> It works for me！You can try

It works for me too~ thanks a lot !

**Comment by yanlongbinluck** [comment]
Hi，what is the environment in Xavier？
Jetpack4.2  is cuda10，ubuntu18.04... by default.
is it ok?
Do we need to configure the following environment on Xavier?
`pytorch 1.0-1.1
ubuntu 1604
TensorRT 5.0
onnx-tensorrt v5.0
cuda 9.0`
Thanks.

**Comment by eagle717** [comment]
@yanlongbinluck cuda10, u18.04  with JP42 is already ok for main branch

**Comment by yanlongbinluck** [comment]
Yeah, Thank you. Longbin YAN 邮箱：yanlongbinluck@gmail.com 签名由 网易邮箱大师 定制 On 06/05/2020 23:41, eagle717 wrote: @yanlongbinluck cuda10, u18.04 with JP42 is already ok for main branch — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by yanlongbinluck** [comment]
> @yanlongbinluck cuda10, u18.04  with JP42 is already ok for main branch

Yeah. Thank you.
